lib/oli/delivery/metrics.ex

```diff
@@ -945,12 +945,12 @@ defmodule Oli.Delivery.Metrics do
            fragment(
              "CAST(COUNT(CASE WHEN ? THEN 1 END) as float) / CAST(COUNT(*) as float)",
              sn.correct
-           )}
+           ), fragment("CAST(COUNT(*) as float)")}
```

Comment:
For v1 analytics (to keep it compatible with the `proficiency_range` function), we use the SQL fragment "CAST(COUNT(*) as float)" to get the total number of first attempts